lib/bookmarker/cli.ex

```diff
@@ -81,7 +81,7 @@ defmodule Bookmarker.CLI do
                             Default: Google Chrome Bookmarks
       -d, --description     Set a description for the rendered markdown.
                             Default: Generated by Bookmarker
-      -or, --order          Set if the list is ordered
+      -s, --s          Set if the list is ordered
```

Comment:
Don't forget the full word "--sort".

![image](https://user-images.githubusercontent.com/9121359/46681684-2898da00-cbc2-11e8-99e3-a53b4698c221.png)

Also don't forget to indent